agent/consul/internal_endpoint.go

```diff
@@ -69,18 +76,67 @@ func (m *Internal) NodeDump(args *structs.DCSpecificRequest,
 		&args.QueryOptions,
 		&reply.QueryMeta,
 		func(ws memdb.WatchSet, state *state.Store) error {
-			index, dump, err := state.NodeDump(ws, &args.EnterpriseMeta, args.PeerName)
-			if err != nil {
-				return err
+			peerings := make([]string, 0)
+			peerings = append(peerings, "") // this represents a local fetch
```

Comment:
```suggestion
			peerings := []string{structs.DefaultPeerKeyword} // this represents a local fetch
```

Reply:
Completely did away with this when we split the local vs peer fetches